### PR TITLE
Display the date of incorporation

### PR DESCRIPTION
**Pull request by richtier**
todo:

 - [x] merge directory api changes too

[this task](https://uktrade.atlassian.net/browse/ED-350)

image

image

image

image

[requires this pr](https://github.com/uktrade/directory-api/pull/67)

### CODE REVIEW

**Comment by konradko** [comment]
@richtier conflicts here

File: enrolment/helpers.py [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+logger = logging.getLogger(__name__)
+
+companies_house_session = requests.Session()
+
+
+def cache_company_details(session, company_number):
+    response = get_companies_house_profile(number=company_number)
+    if not response.ok:
+        response.raise_for_status()
+    session[COMPANIES_HOUSE_PROFILE_SESSION_KEY] = response.json()
```
Comment (richtier) [inline enrolment/helpers.py]: I will update `get_companies_house_profile` to ask for specific fields to avoid making the session cookie huge.

File: enrolment/helpers.py [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+MESSAGE_NETWORK_ERROR = 'A network error occurred'
+COMPANIES_HOUSE_DATE_FORMAT = '%Y-%m-%d'
+COMPANY_PROFILE_URL = 'https://api.companieshouse.gov.uk/company/{number}'
+
+logger = logging.getLogger(__name__)
+
+companies_house_session = requests.Session()
+
+
+def cache_company_details(session, company_number):
```
Comment (konradko) [inline enrolment/helpers.py]: calling storing company details in session 'caching' is a bit misleading, maybe `store_companies_house_profile_in_session`?

File: docker/env.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
       "API_CLIENT_BASE_URL",
       "API_CLIENT_KEY",
       "API_CLIENT_CLASS_NAME",
+      "COMPANIES_HOUSE_API_KEY",
       "COMPANIES_HOUSE_SEARCH_URL",
```
Comment (konradko) [inline docker/env.json]: not part of this change, but this can be a constant in settings - does not change depending on the environment
